OxfordBodleian/Ullendorff/BDLeu1.xml

```diff
@@ -343,9 +343,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <note corresp="#leafdim">Data on leaves dimensions taken from <locus target="#2r"/>.</note>
                                    <!-- <measure type="weight" unit="g"></measure>-->
                                 </extent> 
-                                <foliation>One leaf after <locus target="#135"/> has been skipped in the original foliation and has then been 
+                                <foliation>Pencil foliation in the upper right corner (1–154). One leaf after <locus target="#135"/> 
```

Comment:
Yes, exactly. When the foliation is exactly as expected and can be reconstructed only with the number of folios of the manuscript, it is not noted specifically. Here, I indicated a range because I felt that it was not immediately obvious from reading the next sentence that the foliation ends with 154. What do you think? Should the entire paranthesis be deleted, or changed to (154)? Should the notation in the other manuscripts be adjusted?

Reply:
I think Thea's solution works: it is perhaps redundant to repeat the foliation when it coincides clearly with the total number of folios, but (as Jonas says) it is necessary in all other cases, e.g. foliation/pagination with Roman numbers or mistakes, as in this case.
But if you think that all descriptions should be provided with the foliation range, I'm ok with that: it doesn't take too much space on the page.

Reply:
I've added the folio range everywhere. It will really make the descriptions easier to parse, which is one of our main aims.